tests/unit/test_interface.py

```diff
@@ -116,43 +116,65 @@ def test_set_up_dask_client_default() -> None:
             client = get_client()
 
     set_up_dask_client()
-    client = get_client()
-    assert isinstance(client.cluster, LocalCluster)
-    assert client.cluster.name == "pseudopeople_dask_cluster"
-    workers = client.scheduler_info()["workers"]  # type: ignore[no-untyped-call]
-    assert len(workers) == CPU_COUNT
-    assert all(worker["nthreads"] == 1 for worker in workers.values())
     if is_on_slurm():
         try:
             available_memory = float(os.environ["SLURM_MEM_PER_NODE"]) / 1024
         except KeyError:
             raise RuntimeError(
-                "You are on Slurm but SLURM_MEM_PER_NODE is not set. "
-                "It is likely that you are SSHed onto a node (perhaps using VSCode). "
+                "NOTE: This RuntimeError is expected if you are using VSCode on the cluster!\n\n"
+                "You are on Slurm but SLURM_MEM_PER_NODE is not set; it is likely "
+                "that you are SSHed onto a node (perhaps using VSCode?). "
                 "In this case, dask will assign the total memory of the node to the "
                 "cluster instead of the allocated memory from the srun call. "
                 "Pseudopeople should only be used on Slurm directly on the node "
                 "assigned via an srun (both for pytests as well as actual work)."
             )
     else:
-        available_memory = psutil.virtual_memory().total / (1024 ** 3)
-    assert np.isclose(sum(worker["memory_limit"] / 1024**3 for worker in workers.values()), available_memory, rtol=0.01)
+        available_memory = psutil.virtual_memory().total / (1024**3)
+    _check_cluster(
+        cluster_name="pseudopeople_dask_cluster",
+        memory_limit=available_memory,
+        n_workers=CPU_COUNT,
+        threads_per_worker=1,
+    )
 
 
-def test_set_up_dask_client_custom() -> None:
+def test_set_up_dask_client_existing_cluster() -> None:
+    cluster_name = "custom"
     memory_limit = 1  # gb
     n_workers = 3
+    threads_per_worker = 2
+
+    # Manually create a cluster
     cluster = LocalCluster(  # type: ignore[no-untyped-call]
-        name="custom",
+        name=cluster_name,
         n_workers=n_workers,
-        threads_per_worker=2,
+        threads_per_worker=threads_per_worker,
         memory_limit=memory_limit * 1024**3,
     )
-    client = cluster.get_client()  # type: ignore[no-untyped-call]
+    cluster.get_client()  # type: ignore[no-untyped-call]
+    _check_cluster(cluster_name=cluster_name, memory_limit=memory_limit * n_workers, n_workers=n_workers, threads_per_worker=threads_per_worker)
```

Comment:
Hmm...why isn't black working? This line is 144 characters, far past the 94 character limit set in th pyproject.toml. Not only did black not do anything when I manually ran it but the jenkins build didn't error...